openedx/core/djangoapps/discussions/utils.py

```diff
@@ -31,14 +32,25 @@ def get_divided_discussions(
     divided_course_wide_discussions = []
     divided_inline_discussions = []
 
-    course_wide_discussions = [topic['id'] for __, topic in course.discussion_topics.items()]
-    all_discussions = get_discussion_categories_ids(course, None, include_all=True)
-
-    for divided_discussion_id in discussion_settings.divided_discussions:
-        if divided_discussion_id in course_wide_discussions:
-            divided_course_wide_discussions.append(divided_discussion_id)
-        elif divided_discussion_id in all_discussions:
-            divided_inline_discussions.append(divided_discussion_id)
+    course_key = str(course.id)
```

Comment:
You don't need to cast this to a string I think. It should work directly. 
```suggestion
    course_key = course.id
```